core/templates/dev/head/components/answer_group_editor_directive.html

```diff
@@ -44,7 +44,7 @@
 
 <div style="height: 100%;">
   <!-- For the default answer group, 'rules' is null. Otherwise, 'rules' is a list of rules. -->
-  <div ng-if="rules">
+  <div ng-if="rules" ng-class="stateInteractionIdService.displayed">
```

Comment:
Not a fan of using this because it'll have unintended consequences for other interactions.

Reply:
Is the approach in the other file alright? Should I create a new div if the interaction is Item Selection?

Reply:
Do keep the approaches the same, though I'll defer the specific approach to @seanlip since it looks like extensions are going through a refactor at the moment.